.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,22 +1,19 @@
 ---
-name: Bug Report (.NET Framework)
+name: Bug Report
 about: Create a report to help us improve
 labels: bug
 ---
------------------Bug Template--------------------------------------------------------------
 
-* SDK version: 
-* .NET runtime version (.NET TargetFramework in the .csproj file):
-* Hosting Info (IIS/Azure WebApps/etc): 
-* Platform and OS version: 
+* List of [NuGet packages](https://github.com/microsoft/ApplicationInsights-dotnet#nuget-packages) and version that you are using:
+* Runtime version (e.g. net461, net48, netcoreapp2.1, netcoreapp3.1):
+* Application type (Console/ASP.NET/ASP.NET Core):
+* Hosting environment (e.g. Windows Server 2016 + IIS, Ubuntu 20.04 LTS self-hosted, Azure WebApp):
+  * If you're running the application in Azure Functions, get support [here](TBD).
```

Comment:
How about doing like this? https://github.com/dotnet/aspnetcore/issues/new/choose

Functions can be redirected directly to functions repo.